core/domain/translation_services.py

```diff
@@ -355,6 +359,30 @@ def get_translation_counts(
         ) for entity_translation in entity_translations
     }
 
+def get_translation_counts(
```

Comment:
```
def get_translation_counts(
    entity_type: feconf.TranslatableEntityType,
    entity: exp_domain.Exploration
) -> Dict[str, int]:
```

We already have the above function for exploration. Can we create one generic function to do this?